client/src/feedbacker-components/survey-edit/survey-edit-container.scss

```diff
@@ -13,8 +13,8 @@
 }
 
 .add-button {
-  @include button();
-  background: $palette-light;
+  @extend %anchor-button;
+  background: $palette-white;
```

Comment:
Why are these buttons changed to white? They now look like anchor buttons which they are not

Reply:
no idea

Reply:
@HenrikHelenius Should this be left "as is"  or changed?

Reply:
As the plain style was not like other buttons in the ui, too flat @tobbe-j 

And it's the default button style, there is no special anchor button style, default button and default anchor button should be identical.